oci/layout/oci_archive_transport.go

```diff
@@ -0,0 +1,100 @@
+package layout
+
+import (
+	"github.com/containers/image/docker/reference"
+	"github.com/containers/image/image"
+	"github.com/containers/image/transports"
+	"github.com/containers/image/types"
+)
+
+func init() {
+	transports.Register(ArchiveTransport)
```

Comment:
The `types.ImageTransport` and `types.ImageReference` types, due to their relevance for signature enforcement, should have test coverage close to 100%.  Perhaps roughly base it on `docker/archive/transport_test.go`, or I might be able to help with that later.